src/tools/ScenarioMeasurement/Startup/WindowsTraceSession.cs

```diff
@@ -0,0 +1,77 @@
+﻿using Microsoft.Diagnostics.Tracing;
+using Microsoft.Diagnostics.Tracing.Parsers;
+using Microsoft.Diagnostics.Tracing.Session;
+using System.Collections.Generic;
+using System.IO;
+
+
+namespace ScenarioMeasurement
+{
+    class WindowsTraceSession : ITraceSession
+    {
+        private TraceEventSession session;
+        private Dictionary<TraceSessionManager.KernelKeyword, KernelTraceEventParser.Keywords> winKwMapKernel;
+        private Dictionary<TraceSessionManager.ClrKeyword, ClrPrivateTraceEventParser.Keywords> winKwMapClr;
+
+        public WindowsTraceSession(string sessionName, string traceName, string traceDirectory, Logger logger)
+        {
+            string fileName = Path.ChangeExtension(traceName, "perflab.etl");
+            string filePath = Path.Combine(traceDirectory, fileName);
+            // Currently kernel and user share the same session
+            session = new TraceEventSession(sessionName, filePath);
+            InitWindowsKeywordMaps();
```

Comment:
I believe we decided to split these apart, yes?

Reply:
Yes. Update is coming soon.